sdk/resourcemanager/azure-resourcemanager-sql/src/main/java/com/azure/resourcemanager/sql/models/DatabaseSku.java

```diff
@@ -0,0 +1,344 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.sql.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import java.util.Collection;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+/** Defines SQL Database Sku. */
+@Fluent
+public final class DatabaseSku {
+    private static final ConcurrentMap<String, DatabaseSku> VALUES = new ConcurrentHashMap<>();
+
+    /** System Edition with System sku. */
+    public static final DatabaseSku SYSTEM_SYSTEM = new DatabaseSku("System", "System", null, 0, null);
```

Comment:
Just curious, does anyone has a non-`null` size?

Reply:
It seems currently no.